src/main/java/com/commercetools/pingpong/model/Game.java

```diff
@@ -91,5 +126,67 @@ public Player getPlayerTwo() {
         return playerTwo;
     }
 
-}
+    public boolean getPlayerOneServe() {return playerOne.amIServing();}
+
+    public boolean getPlayerTwoServe() {return playerTwo.amIServing();}
+
+    public boolean isOvertime(Player playerOne, Player playerTwo) {
+        return playerOne.getSetScore() >= 10 && playerTwo.getSetScore() >= 10;
+    }
+
+    public void whoServesAfterDoubleClick() {
+        if (!isSetScoreEven()) {
+            if (playerOne.amIServing()) {
+                playerOne.changeServe();
+                playerTwo.setServe();
+
+            } else if (playerTwo.amIServing()) {
+                playerTwo.changeServe();
+                playerOne.setServe();
+            }
+        }
+    }
 
+    public void setFirstServe(Player actingPlayer) {
+        actingPlayer.setBeginningServe();
+        actingPlayer.setServe();
+        }
```

Comment:
This bracket is probably wrongly indented.

Reply:
sorry ☺️ 
06b114eaf36bb234810d9ecf3eb6cd26186d2197